framework/src/application/node/transport/schemas.ts

```diff
@@ -34,7 +34,6 @@ export const schemas = {
 					type: 'string',
 					format: 'base64',
 				},
-				minItems: 1,
```

Comment:
shouldn't this be kept if it's expecting an array of transactionIds?

Reply:
ah sorry this was part of #5484 (wrong target branch)